django/contrib/admin/checks.py

```diff
@@ -18,9 +18,13 @@
 
 
 def check_admin_app(**kwargs):
-    from django.contrib.admin.sites import system_check_errors
+    from django.contrib.admin.sites import all_sites
```

Comment:
My sense is that these checks should be skipped if the admin isn't in the `app_configs` kwarg.
```python
if app_configs and not any(app_config.name == 'django.contrib.admin' for app_config in app_configs):
    return []
```
I think that fixes some of the test isolation issues. @charettes?